tests/e2e/workloads/app/test_jenkins_simulation.py

```diff
@@ -54,7 +54,7 @@ def test_git_clone(self, pod):
         relevant_pod_logs = None
         func_calls = ["NodeStageVolume", "NodeGetVolumeStats"]
         error_msg = "System has not been booted with systemd"
-
+        inode_info = '"unit":2'
```

Comment:
add bugzilla marker

Reply:
Cant add 2 BZ marker. added comment "Test coverage for BZ 2132270"

Reply:
also bz marker are linked to our bugzilla and tests only run if these bzs are verified so its important to add

Reply:
Okay.

Reply:
Done